### ISSUE TITLE
For tabs, using the arrow keys to navigate between tabs doesn't work reliably. Sometimes you can't tab to or arrow back to the previous tabs.

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2051]
# Bug Report

## Description
For tabs, using the arrow keys to navigate between tabs doesn't work reliably. Sometimes you can't tab to or arrow back to the previous tabs.

## Steps to Reproduce

1. TBD

## Additional Context / Screenshots
Identified as part of functional verification on [this PR](https://github.com/cerner/terra-core/pull/2004#issuecomment-438396868).

## Expected Behavior
Using the arrow keys to navigate between tabs works reliably.

## Environment

* Component Name and Version: terra-tabs
* Browser Name and Version: IE 11 + JAWS

**Comment by JakeLaCombe** [comment]
I talked to @mmalaker on this, and we will have limitations on this Jira. In JAWS, using the left and right arrow keys cycles through the individual letters on a given text node. I don't think it's desirable to override this behavior for a screen reader.

As for actionable changes, I noticed that JAWS was working great with the up and down arrows, so we could use up and down navigation as the requirements for cycling through tabs. In addition, without screen reader support, only the navigation is tabbable as opposed to the individual tabs. We should probably add tabindex="0" to the tabs that are not disabled.
